src/main/java/emcshop/scraper/PlayerProfile.java

```diff
@@ -10,13 +10,13 @@
  */
 public class PlayerProfile {
     private final String playerName, portraitUrl, title;
-    private final boolean private_;
+    private final boolean privateRank;
```

Comment:
Why was this renamed to "privateRank"?  This field defines whether the player's profile is private or not.  If a profile is private, it can only be viewed if you are logged into a forum account.  The person also has to be following you, if they have set this in their privacy settings.